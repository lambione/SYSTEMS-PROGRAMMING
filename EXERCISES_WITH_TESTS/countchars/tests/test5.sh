#!/bin/sh
exec "$1" cba <<EOF
some good charachters here.
only some!
EOF
