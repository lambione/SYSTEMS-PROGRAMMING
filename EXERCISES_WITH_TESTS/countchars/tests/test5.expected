(stdin) c:2 b:0 a:2
